src/main/java/com/meronat/iplog/IPLog.java

```diff
@@ -185,10 +192,21 @@ public Logger getLogger() {
 
     }
 
+    public PluginContainer getPluginContainer() {
```

Comment:
I find it in good practice to have the PluginContainer available, because someone once told me that it was better practice to pass a PluginContainer to all the sponge methods with a parameter of Object plugin. As far as I remember he said that passing something else would just lead to a map lookup for the PluginContainer. Because of this I like to have it available for things like passing Cause etc. Not necessary, you can remove if you want.